cabal-install/Distribution/Client/Init/Types.hs

```diff
@@ -78,12 +78,14 @@ data InitFlags =
   -- ones, which is why we want Maybe [foo] for collecting foo values,
   -- not Flag [foo].
 
-data PackageType = Library | Executable
+data BuildType = LibBuild | ExecBuild
+
+data PackageType = Library | Executable | LibraryAndExecutable
   deriving (Show, Read, Eq)
 
 instance Text PackageType where
   disp = Disp.text . show
```

Comment:
Maybe make the `Text` instance display `LibraryAndExecutable` as "Library and Executable" so that it'd be shown like that in the menu? Though you'd also need to fix the `parse` function (unless we don't actually use it, then you can just get rid of the `Text` instance and make `disp` a standalone function).

Reply:
Done in 2b835e662cbc9cfb7dd6dea2ed5399409a566f6f.